woocommerce/woocommerce-ios
# Add taxes to receipts

Resolves #4437 

## Description
Adds taxes to the receipt, where these are non-zero.
This implementation adds only the tax total, ignoring shipping tax and discount tax which are separately available. I initially included these in the receipt, but found that we don't split them out in the Cart, Checkout, or Confirmation on the web. Particularly since we don't have the items tax total, I think that adding just some of the components would add confusion, as the tax total wouldn't add up to the tax components that we print.

`ReceiptCartTotals` is a bit overkill at the moment, but will be used for more fields in #4438, #4439, and #4440.

## Testing
To test, an In Person Payment setup is required. See PdfdoF-D-p2 for info.

1. Go to the Orders tab
2. Open an unpaid "Pay on Delivery" order with non-zero taxes
3. Take card payment
4. Tap "Print Receipt"
5. Observe that the order lines include a "Total Tax" line.
6. Repeat with an order that has zero tax, observe that taxes are not shown.

![Simulator Screen Shot - iPhone 12 mini - 2021-09-30 at 17 06 30](https://user-images.githubusercontent.com/2472348/135495737-cedda1c1-7038-442e-96c1-d648540b9766.jpg)



Update release notes:
